requirements.txt

```diff
@@ -20,6 +22,7 @@ screeninfo
 skia-pathops
 svgelements>=1.8.1
 sympy
+tempfile
```

Comment:
@3b1b This new line got me into some trouble and I couldn't install Manim anymore. I.e. `pip install -r requirements.txt` results in this error:

```
ERROR: Could not find a version that satisfies the requirement tempfile (from versions: none)
ERROR: No matching distribution found for tempfile
```

Maybe it's related to [this issue](https://stackoverflow.com/questions/44286628/could-not-find-a-version-that-satisfies-the-requirement-backports-tempfile-from)? For now, I removed this line locally to get Manim up and running again.

---

Note that originally I just ran `pip install -e .` as usual but that didn't install the new `appdirs` library and therefore Manim failed on startup. Isn't `pip install -e .` supposed to install the required dependencies?

Reply:
Also when searching on [Pip](https://pypi.org/search/?q=tempfile), I can't seem to find any library that is called `tempfile`.

Reply:
I may have neglected to update the requirement.txt, I'll take a look shortly.

Reply:
Hello :-)
You definitely need a CI for testing and showing for all users/dev how to install properly Manim and dependencies (OS/system dependencies).
I can (easily) do this for Linux (/Debian) OS, more difficult for me right now to write this workflow for Windows and MacOS platform (but others can replicate and adapt my attempt for Linux platform).
Btw very interesting project, i like it a lot :-)